tests/repositories/parsers/test_pypi_search_parser.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+
+from poetry.repositories.parsers.pypi_search_parser import Result
+from poetry.repositories.parsers.pypi_search_parser import SearchResultParser
+
+
+FIXTURES_DIRECTORY = Path(__file__).parent.parent / "fixtures" / "pypi.org" / "search"
```

Comment:
You could use `fixture_dir: FixtureDirGetter` to achieve the same

Reply:
Can't really, cause this goes 2 `.parent` up, and `fixture_dir` goes only 1 `.parent` up.

Reply:
Ah, these are not `tests/fixtures` but `tests/repositories/fixtures`. Then, it's just that...